guacamole/src/main/frontend/src/app/client/controllers/clientController.js

```diff
@@ -26,6 +26,7 @@ angular.module('client').controller('clientController', ['$scope', '$routeParams
     // Required types
     var ConnectionGroup    = $injector.get('ConnectionGroup');
     var ManagedClient      = $injector.get('ManagedClient');
+    var ManagedClientGroup = $injector.get('ManagedClientGroup');
```

Comment:
What do you think of the new `let` and `const` keywords? I'm certainly not asking you to rewrite this beast, but I find them to be clearer than using `var`. 

Plus we'll get compile-time checking for things that shouldn't be mutated.

Reply:
Sure. I'll at least update the files that have been touched here.

Reply:
OK - migrated all directly touched code from `var` to `const`/`let` via rebase. I avoided making sweeping changes, updating only the code that is already within this diff.